changelog/unreleased/issue-2277

```diff
@@ -0,0 +1,6 @@
+Enhancement: Add support for ppc64le
+
+    Adds support for ppc64le, the processor
+    architecture from IBM.
```

Comment:
I'm sorry if you feel I'm nitpicking, but this text shouldn't be indented :)

Reply:
done!